### PR TITLE
add checking available miniapp  before showing downloaded minapp list

### PR DESCRIPTION
**Pull request by munir-rakuten**
# Description
Deleted app from Ras portal which is also removed from mini app list

## Links
MINI-2907

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/permission/MiniAppDownloadedListActivity.kt [hunk 1]
```diff
@@ -63,16 +76,41 @@ class MiniAppDownloadedListActivity(private val miniApp: MiniApp) : BaseActivity
...
+            if (availableMiniAppList.contains(it.first)) {
+                miniAppList.add(it.first)
+                miniAppPermissions.add(parsePermissionText(it.second.pairValues))
+            }
         }
         adapter.addDownloadedList(miniAppList, miniAppPermissions)
         updateEmptyView()
     }
 
+    private fun executeLoadingList() {
```
Comment (khairul-alam-licon) [inline testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/permission/MiniAppDownloadedListActivity.kt:110]: Maybe we can rename to `loadAvailableList` or just a cleanup e.g.

```
    override fun onStart() {
        ...
        viewModel.getMiniAppList()
    }
```

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/permission/MiniAppDownloadedListActivity.kt [hunk 2]
```diff
@@ -63,16 +76,41 @@ class MiniAppDownloadedListActivity(private val miniApp: MiniApp) : BaseActivity
...
+                        checkAndUpdateList(miniAppListAvailable)
+                    })
+                errorData.observe(this@MiniAppDownloadedListActivity, Observer {
+                    val list = MiniAppListStore.instance.getMiniAppList()
+                    if (list.isNotEmpty()) {
+                        checkAndUpdateList(list)
+                    }
+                })
+            }
+    }
```
Comment (khairul-alam-licon) [inline testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/permission/MiniAppDownloadedListActivity.kt:95]: We can put a blank line after this..

**Review by khairul-alam-licon — APPROVED** [review]
Looks good to go 👍 
I left couple of minor suggestions.

**Review by minh-rakuten — APPROVED** [review]
👍 👍

**Pull request merged**